src/helpers/get-linked-pull-requests.ts

```diff
@@ -12,37 +11,31 @@ export async function getLinkedPullRequests(
   context: Context,
   { owner, repository, issue }: GetLinkedParams
 ): Promise<GetLinkedResults[]> {
-  const logger = context.logger;
+  if (!issue) return [];
+  // const logger = context.logger;
   const collection = [] as GetLinkedResults[];
-  const { data } = await axios.get(`https://github.com/${owner}/${repository}/issues/${issue}`);
-  const dom = parse(data);
-  const devForm = dom.querySelector("[data-target='create-branch.developmentForm']") as HTMLElement;
-  const linkedList = devForm.querySelectorAll(".my-1");
-  if (linkedList.length === 0) {
-    context.logger.info(`No linked pull requests found`);
-    return [];
-  }
-
-  for (const linked of linkedList) {
-    const relativeHref = linked.querySelector("a")?.attrs?.href;
-    if (!relativeHref) continue;
-    const parts = relativeHref.split("/");
-
-    // check if array size is at least 4
-    if (parts.length < 4) continue;
+  const pulls = await getAllPullRequests(context);
+  const currentIssue = await context.octokit.issues.get({
```

Comment:
Hope this instance is authenticated?

Reply:
Yes. We are using app's octokit who has permission to the private repos